shared/utils/yaml-to-shorthand.py

```diff
@@ -45,6 +37,14 @@ def add_sub_element(parent, tag, data):
     return element
 
 
+def add_warning_elements(element, warnings):
+    for warning_dict in warnings:
+        warning = add_sub_element(element, "warning", warning_dict.values()[0])
+        warning.set("category", warning_dict.keys()[0])
+
+    return element
```

Comment:
no need to return

Reply:
I get an error as it "returns" `None` if the return doesn't exist.